docs/operations/password-provider.md

```diff
@@ -33,7 +33,10 @@ Environment variable password provider provides password by looking at specified
 e.g
 
 ```json
-{ "type": "environment", "variable": "METADATA_STORAGE_PASSWORD" }
+druid.metadata.storage.type=mysql
```

Comment:
password provider can also be use in other places outside of the config files too. For example, the ingestionSpec. I think it's irrelevant to mention ...metadata.storage... stuff here